README.md

```diff
@@ -44,10 +71,15 @@ An `nvmrc` file is included if using [nvm](https://github.com/creationix/nvm).
 
 From within the root directory:
 
+1. Install webpack.
+2. Install project dependencies.
```

Comment:
If someone installs the project dependencies, do they need to install webpack separately?